kin-ecosystem-sdk/src/main/java/com/kin/ecosystem/data/model/WhitelistData.java

```diff
@@ -0,0 +1,54 @@
+package com.kin.ecosystem.data.model;
+
+public class WhitelistData {
+
+    private String userID;
+    private String appID;
+    private String apiKey;
+
+    public WhitelistData(String userID, String appID, String apiKey) {
+        this.userID = userID;
+        this.appID = appID;
+        this.apiKey = apiKey;
+    }
+
+    public String getUserID() {
+        return userID;
+    }
+
+    public String getAppID() {
+        return appID;
+    }
+
+    public String getApiKey() {
+        return apiKey;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+
+        WhitelistData that = (WhitelistData) o;
+
+        if (userID != null ? !userID.equals(that.userID) : that.userID != null) {
```

Comment:
if userID != null && userID.equals(that.userID) 

or Objects.equals()?

Reply:
we can't use Objects.equals()

Reply:
why not using && if userID!=null checking !userID.equals(that.userID) is enough

Reply:
it's generated from IntelliJ and I didn't change that, is that so important?